vidify/player/web.py

```diff
@@ -0,0 +1,83 @@
+"""
+"""
+
+import logging
+from typing import Optional
+from threading import Thread
+from webbrowser import open
+from threading import Timer, Thread
+
+try:
+    import flask
+except OSError:
+    raise ImportError("Flask is not installed")
+
+from vidify import Platform, CURRENT_PLATFORM
+from vidify.player.generic import PlayerBase
+from PyQt5.QtCore import QObject
+import sys
+import os
+
+PORT = 9999
+ROOT_URL = 'http://localhost:{}/video'.format(PORT)
+DEFAULT_VIDEO_ID = "GfKs8oNP9m8"
+class WebPlayer(PlayerBase, QObject):
+    DIRECT_URL = False
+    app = None
+    currentMedia = ""
```

Comment:
Don't forget to use camelcase variables for Python code, this should be `current_media`. Also aplies to `isPlaying` and maybe others.

Reply:
Also, perhaps you should use `None` for an empty `currentMedia` rather than an empty string.

Reply:
the first suggestion isn't resolved yet, by the way

Reply:
I think this hasn't updated but I think I've made the changes now :)